flow/scripts/synth_preamble.tcl

```diff
@@ -55,16 +55,6 @@ if {[info exist ::env(PRESERVE_CELLS)]} {
   }
 }
 
-
-
-if {[info exist ::env(BLOCKS)]} {
-  hierarchy -check -top $::env(DESIGN_NAME)
```

Comment:
This is not really a check, it is just making sure that the hierarchy is evaluated so that blackbox can be run.

synth_preamble.tcl is trying to leave as much processing for synth.tcl as possible, near as I can tell.

This check is still being run, only later, as is the intent of synth_preamble/synth.tcl, near as I can tell.